packages/palette/src/elements/Dialog/__tests__/Dialog.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { mount } from "enzyme"
+import React from "react"
+import { Dialog } from "../Dialog"
+
+describe("Dialog", () => {
+  it("displays custom title", () => {
+    const primaryCtaAction = () => null
+    const component = mount(
+      <Dialog
+        primaryCta={{
+          action: () => ({ primaryCtaAction }),
+          text: "Continue",
+        }}
+        title="Condimentum Ridiculus"
+        show
+      />
+    )
+    expect(component.html()).toContain("Condimentum Ridiculus")
+  })
+})
```

Comment:
Can we add a few more tests that cover all of the different behavior? 
- `primaryCta` / secondary actions and labels
- showing / hiding 
- dismissal

Reply:
@damassi - totally. I mainly PR'ed this in current state to get some feedback on the idea of reusing `Modal` with a very predefined values for `Dialog` implementation. Was not sure if it'll be better to go this way or try to abstract needed classes and then include them in both `Modal` and `Dialog` implementation as it was done in `Reaction`. 

I think the code looks simpler with `Dialog` using `Modal` directly and don't see any major drawbacks with  going this way, but maybe I'm missing something here?

Reply:
No this is great! The design looks super solid / ideal 👍